modules/gke-cluster/variables.tf

```
@@ -203,3 +203,9 @@ variable "enable_vertical_pod_autoscaling" {
   type        = string
   default     = false
 }
+
+variable "services_secondary_range_name" {
+  description = "The name of the secondary range within the subnetwork for the services to use"
+  type        = string
+  default     = ""
```

```suggestion
  default     = null
```
Let's change this to: `null` instead.